planner/optimizer/.gitignore

```diff
@@ -4,5 +4,4 @@ target/
 .classpath
 .project
 .metadata
-.settings/
-src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/output*
```

Comment:
if this line "src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/output*" is removed, but the output files of the test in OptimizerTest.java are still generated in 

private static final String OUTPUT_FILENAME       = "./src/test/java/eu/seaclouds/platform/planner/optimizerTest/resources/target/output";

I think that the output files will continue being included in the commits.